src/nextcloud/talk/recording/Participant.py

```diff
@@ -218,6 +219,7 @@ def startChrome(self, width, height, env):
         :param height: the height of the browser window.
         :param env: the environment variables, including the display to start
                     the browser in.
+        :param executable_path: the direct path to the chromium executable
```

Comment:
_executable_path_ is not the path to the _chromium_ executable, but to the _chromedriver_ executable